src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -456,3 +482,11 @@ class SupportNotFoundException(message: String? = null, cause: Throwable? = null
     return SupportNotFoundException(message, cause)
   }
 }
+class ExpiredVisitAmendException(message: String? = null, cause: Throwable? = null) :
```

Comment:
do we put our exceptions in side the class, what convention are we using on this project

Reply:
So far all exceptions are being added this way (in VisitService and SessionTemplateService).